package.json

```diff
@@ -50,6 +50,7 @@
     "druid.d.ts": "^0.12.1",
     "has-own-prop": "^1.0.1",
     "immutable-class": "^0.11.1",
+    "is-ip": "3.1.0",
```

Comment:
This should use the latest. Please elaborate on why it didn't work

Reply:
why do we need the `is-ip` dependency at all? I say get rid of it and treat `{ip:'500.600.700.800'}` as a valid IP type. Fewer dependancies is better and in this case the reason we are adding the IP type is to pick the best DB function to use. I don't think we need to work that hard to make sure we only have valid IPs. Maybe just do a best effort regexp like `/^[\d.:]+$/` as a ckeck.